cypress/e2e/court-cases/court-case-details/CourtCaseDetails.cy.ts

```diff
@@ -382,40 +451,6 @@ describe("Court case details", () => {
     cy.get(".moj-tab-panel-triggers").contains("There are no triggers for this case.")
   })
 
-  it("should select all triggers when select all link is clicked", () => {
```

Comment:
I've moved the trigger tests in this file alongside the new trigger tests in their own file